apps/web/src/services/asset-inventory/collector/CollectorPage.vue

```diff
@@ -18,16 +18,18 @@
                 </router-link>
             </template>
         </p-heading>
+        <!-- FIXME: loading must be fixed after basic function. -->
         <p-data-loader
-            :data="cloudCollectorPageState.collectorList"
-            :loading="state.loading && !cloudCollectorPageState.collectorList"
+            :data="collectorPageState.collectorList"
+            :loading="collectorPageState.loading && !collectorPageState.collectorList"
```

Comment:
Question: I wonder why checking `loading` and `collectorList` together by `and`.

Reply:
On the collector landing page, there are two loaders: one for loading when there is no unfiltered list, and another for loading during filtering. I used '&&' to differentiate between them! However, this part may change in the future when improving the p-data-loader.

Reply:
Thank you for answer!